lua/lspconfig/configs/turbo_ls.lua

```diff
@@ -0,0 +1,26 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'turbo-language-server', '--stdio' },
+    filetypes = { 'html', 'ruby', 'eruby', 'blade', 'php' },
+    root_dir = util.root_pattern('Gemfile', '.git'),
```

Comment:
you can use `vim.fs.root` here

Reply:
Thanks, @glepnir; I did not know about `vim.fs.root()`.